docs/developer-docs/latest/development/backend-customization/requests-responses.md

```diff
@@ -19,3 +19,55 @@ For more information, please refer to the [Koa request documentation](http://koa
 The context object (`ctx`) contains a list of values and functions useful to manage server responses. They are accessible through `ctx.response`, from [controllers](/developer-docs/latest/development/backend-customization/controllers.md) and [policies](/developer-docs/latest/development/backend-customization/policies.md).
 
 For more information, please refer to the [Koa response documentation](http://koajs.com/#response).
+
+## Accessing the request context anywhere
+
+_**Added in:** v4.3.9_
+
+Strapi exposes a way to access the current request context from anywhere in the code. (e.g lifecycle functions)
+
+You can access the request as follows:
+
+```js
+const ctx = strapi.requestContext.get();
+```
+
+You should only use this inside of functions that will be called in the context of an HTTP request.
+
+```js
+// correct
+
+const service = {
+  myFunction() {
+    const ctx = strapi.requestContext.get();
+    console.log(ctx.state.user);
+  },
+};
+
+// incorrect
+const ctx = strapi.requestContext.get();
+
+const service = {
+  myFunction() {
+    console.log(ctx.state.user);
+  },
+};
+```
+
+**Example:**
+
+```js
+//path: ./api/test/content-types/article/lifecycles.js
+
+module.exports = {
+  beforeUpdate() {
+    const ctx = strapi.requestContext.get();
+
+    console.log('User info in service: ', ctx.state.user);
+  },
+};
+```
+
+::: tip
```

Comment:
```suggestion
::: note
```

It's rather a note as we don't provide specific instructions or advice that could be used :)